src/css/typography.sass

```diff
@@ -66,3 +66,20 @@ h6
 .text-caption
   font-size: 1.2rem
   font-family: "ralewayitalic"
+
+.wallet-btn
+  border: 0.5px solid #cccccc
+  font-family: 'ralewayregular'
+  border-radius: 6px 6px 6px 6px
+
+.text-wallet-heading
```

Comment:
At a later stage we might need to refactor the `.text-wallet-*` classes into something that is general enough for the entire application, but for now this is fine